core/templates/dev/head/profile/Profile.js

```diff
@@ -19,15 +19,21 @@
  */
 
 oppia.controller('Profile', [
-  '$scope', '$http', '$rootScope',
-  function($scope, $http, $rootScope) {
+  '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
+  function($scope, $http, $rootScope, $oppiaDatetimeFormatter) {
```

Comment:
There should be no $ here. I'm actually surprised that getLocaleAbbreviatedDatetimeString still works; does it not produce a console error?